src/main/scala/org/scalacheck/ops/time/JavaTimeGenerators.scala

```diff
@@ -0,0 +1,46 @@
+package org.scalacheck.ops.time
+
+import java.time.temporal._
+import java.time.{Duration, Instant}
+
+object JavaTimeGenerators extends JavaTimeGenerators
+trait JavaTimeGenerators extends AbstractDateTimeGenerators {
+  override type DateTimeType = Instant
+  override type DurationType = TemporalAmount
+  override type ParamsType = JavaTimeParams
```

Comment:
In the future, I want to split these types up better. All time libraries have durations and instants, but some have multiple types of durations and only one type of instant and others have multiple types of instants. So I'm thinking of separating this class into 2 classes, one for instants and one for durations. The only problem is that there are some overlap because some methods work by subtracting durations from instants, however, I think I can just map instants into `Long`s and then do the math that way.